_posts/2020-09-21-centralised-schema-management-microservices.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: Centralized schemas and microservices, match made in Hell?
+author: vehvis
+excerpt: >
+  Schemas: love 'em or hate 'em, can't live without 'em -- not even when you're doing microservices. In this post I describe
+  how we centralized our schema management into a (hopefully) long-term solution, all the while keeping a tenuous hold of our sanity.
+tags:
+ - Microservices
+ - Clojure
+ - Development
+ - Architecture
+---
+
```

Comment:
You mention XML a couple of times, but you don't mention Ontologies in any way. Aren't they pretty much the same thing you've implemented? Common evolvable representations of shared data? They were present already in the last millennium, I believe, so maybe explain a bit how far we have come from there? Or am I missing something?